bazel/README.md

```diff
@@ -721,13 +721,18 @@ https://github.com/bazelbuild/bazel/issues/2805.
 
 # Coverage builds
 
-To generate coverage results, make sure you are using a clang toolchain and have `llvm-cov` and
+To generate coverage results, make sure you are using a Clang toolchain and have `llvm-cov` and
 `llvm-profdata` in your `PATH`. Then run:
 
 ```
 test/run_envoy_bazel_coverage.sh
 ```
 
+**Note** that it is important to ensure that the versions of `clang`, `llvm-cov` and `llvm-profdata`
```

Comment:
llvm-profdata is set in .bazelrc. for me, i have to manually update to a full path or make sure that the PATH bazel action env is pointing to the llvm-profdata of the right version.

May be worth mentioning that llvm-profdata is references in bazelrc, or adding a similar check in the script if possible.

Reply:
If you set PATH correctly I don't think you need to set full path there?

Reply:
Correct, I initially was setting it explicitly, but I think my real issue boiled down to using an old version of Clang/LLVM (9.0 instead of 10.0), as well as relying on explicit CC/CXX rather than path.

Unfotunately, `llvm-profdata` doesn't have a version flag, so I think the best thing to do is assume that since we're querying `llvm-cov` from the path and it's likely coresident with `llvm-profdata`, it provides a suitable surrogate.

Reply:
@htuch llvm-profdata have a version flag but need a subcommand first, i.e. `llvm-profdata show --version` does print version info.

Reply:
> If you set PATH correctly I don't think you need to set full path there?

Ah, probably. I think I confounded two solutions (running setup_clang.sh to get PATH correctly OR full reference for llvm-protfdata)